Rock/Workflow/Action/CheckIn/FilterGroupsByAge.cs

```diff
@@ -18,24 +18,23 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.ComponentModel.Composition;
-using System.Globalization;
 using System.Linq;
 
 using Rock.Attribute;
 using Rock.Data;
+using Rock.Web.Cache;
 
 namespace Rock.Workflow.Action.CheckIn
 {
     /// <summary>
     /// Removes (or excludes) the groups for each selected family member that are not specific to their age.
     /// </summary>
-    [ActionCategory( "Check-In" )]
     [Description( "Removes (or excludes) the groups for each selected family member that are not specific to their age." )]
```

Comment:
Strange!  I can't find it being removed in any of our recent commits.  I added it back.